LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ILoraModuleClient.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer.BasicsStation.ModuleConnection
+{
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Shared;
+    using System;
+    using System.Threading.Tasks;
+
+    public interface ILoraModuleClient:IAsyncDisposable
+    {
+        uint OperationTimeoutInMilliseconds { get; set; }
```

Comment:
Can we use `TimeSpan` here instead and name it simply `OperationTimeout`? Seems simpler and more strong-typed to me.